backend/clients/twilio_client.py

```diff
@@ -2,16 +2,32 @@
 import json
 import logging
 import time
+import phonenumbers
+
+from datetime import datetime
 from json.decoder import JSONDecodeError
 from twilio.base.exceptions import TwilioRestException
-
 from twilio.rest import Client
+from pydantic import BaseModel, ValidationError
+from pydantic_extra_types.phone_numbers import PhoneNumber
+
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
-class TwiliobotClient:
+class IncomingMessage(BaseModel):
+    MessageSid: str
+    From: str
```

Comment:
Can this be a phone number type?

Reply:
We can't use phone number there because we receive `whatsapp:+phonenumber` format, but we validate the phone number when we format the message into queue message here: https://github.com/akvo/rag-doll/blob/c9d21e11ffb86adf3d6595dbcd52e3b5392be2f1/backend/routes/twilio_routes.py#L28-L34

And this is the format queue message function:
https://github.com/akvo/rag-doll/blob/c9d21e11ffb86adf3d6595dbcd52e3b5392be2f1/backend/clients/twilio_client.py#L107-L136

Reply:
Hi @kjkoster 
And I also want to confirm, I think we should use below format as the twilio `format_to_queue_message`. Is that correct?
https://github.com/akvo/rag-doll/blob/fef8bc0b0f0307da31638f4f1a9167a5e58de010/packages/Akvo_rabbitmq_client/src/Akvo_rabbitmq_client/queue_message_util.py#L4-L42

Reply:
Looks mostly good. Can you make `client_phone_number` and `user_phone_number` into the phone number type?

Can you make the platform into an `enum`?

What is the `headers` field for?

Reply:
Base on this [comment](https://github.com/akvo/rag-doll/pull/28#discussion_r1665154989), so the headers will removed. I have plan to add headers so we can add the `reply_to` params there.